Automattic/wp-calypso
# Decouple newsletter subscription from tier creation

Related to https://github.com/Automattic/gold/issues/190
Blocked by https://github.com/Automattic/wp-calypso/pull/83433 and 

## Proposed Changes

* Decouple newsletter subscription from tier creation


## Testing Instructions



* Open the calypso live link and you should see those panels:

<img width="400" alt="Screenshot 2023-10-25 at 14 31 21" src="https://github.com/Automattic/wp-calypso/assets/790558/9bc63825-3217-4a40-9891-0c2110b991ac">
<img width="400" alt="Screenshot 2023-10-25 at 14 31 27" src="https://github.com/Automattic/wp-calypso/assets/790558/3e92e49e-525c-4cc2-b972-3394f9c9c64b">
<img width="400" alt="Screenshot 2023-10-25 at 14 31 52" src="https://github.com/Automattic/wp-calypso/assets/790558/9b557f44-af06-4b07-b4d2-38bf8bd59b5c">



* Make sure plans are created as expected with 
```
print_r(Memberships_Product::get_product_list(<your blog id>)

```
